arealite/api/llm_client_api.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2025 Ant Group Inc.
+# Licensed under the Apache License, Version 2.0
+
+import abc
+import asyncio
```

Comment:
I saw there is `uvloop` as the async manager. Does it actually used in the project